lib/propolis/src/common.rs

```diff
@@ -11,13 +11,25 @@ use crate::vmm::SubMapping;
 
 /// Controls whether items wrapped in a [`GuestData`] are displayed or redacted
 /// when the wrappers are printed via their `Display` or `Debug` impls.
+#[no_mangle]
```

Comment:
(definitely worth a note like `// The #[no_mangle] here is not for internal correctness, but to ensure that two cohabitant lib-propolis versions produce an error rather than ambiguously work`)

Reply:
Done in 7334b8d.